bal/blockchain.py

```diff
@@ -169,9 +224,11 @@ def do_register_peers():
     simulation_path = args.simulationpath
     if simulation_path != '':
         if args.variant.find('pos') == 0:
-            blockchain = POSBlockchainSimulation(p2p_port, initial_difficulty, simulation_path, args.name)
+            blockchain = POSBlockchainSimulation(
+                p2p_port, initial_difficulty, simulation_path, args.name)
         else:
-            blockchain = POWBlockchainSimulation(p2p_port, initial_difficulty, simulation_path, args.name)
+            blockchain = POWBlockchainSimulation(
```

Comment:
@UkcaGreen why there is a new line right after '(' ? if it's because of line length, is it also possible to write it as 
`POWBlockchainSimulation(
    p2p_port,
    initial_difficulty,
    simulation_path,
    args.name)` ?